src/components/UniversalTable/UniversalTable.module.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../app/variables.scss";
+
+.TableContainer {
+  margin: 100px auto;
+  color: $tableMain;
+  width: $contentContainerWidth;
+}
+
+.TableHeader {
+  position: relative;
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+  z-index: 1;
+  box-shadow: 0px 3px 3px rgba(0, 0, 0, 0.458);
+
+  &:nth-child(odd) {
+    background-color: $lightGray;
+  }
+}
+
+.TableRow {
```

Comment:
why would you name your classes with a capital letter?

Reply:
solved